src/char/char.hpp

```diff
@@ -14,19 +14,30 @@
 
 #include "packets.hpp"
 
+using namespace rathena::server_core;
+
+namespace rathena{
+	namespace server_character{
```

Comment:
just make this `namespace rathena::server_character`

also, can we choose not to indent in namespaces? Otherwise we gonna get wiiiiiiiiiiiiiiide

Reply:
As discussed on Discord - not yet possible.